gtsam_unstable/gtsam_unstable.i

```diff
@@ -567,6 +567,10 @@ virtual class FixedLagSmoother {
   double smootherLag() const;
 
   gtsam::FixedLagSmootherResult update(const gtsam::NonlinearFactorGraph& newFactors, const gtsam::Values& newTheta, const gtsam::FixedLagSmootherKeyTimestampMap& timestamps);
+  gtsam::FixedLagSmootherResult update(const gtsam::NonlinearFactorGraph &newFactors,
```

Comment:
You can just update the above `update` by adding the extra parameter and specifying a default parameter.